### PR TITLE
[IMPROVEMENT] Improvements on the App Icon selection

### LINKED ISSUE #1285: [IMPROVEMENT] Change text displayed to user on changing icon
https://github.com/RocketChat/Rocket.Chat.iOS/blob/2b2707ed9059b6deb34b364204321d3ec30a4a9d/Rocket.Chat/Resources/en.lproj/Localizable.strings#L203

This text is quite long and not grammatically correct. I suggest changing the text to something like 'Choose an icon'

### PR DESCRIPTION
**Pull request by rafaelks**
@RocketChat/ios

Closes #1285

### CODE REVIEW

**Comment by artrmz** [comment]
Looking great!

File: Rocket.Chat/Views/Cells/Preferences/ReusableViewText.swift [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+//
+//  Created by Rafael Kellermann Streit on 07/03/18.
+//  Copyright © 2018 Rocket.Chat. All rights reserved.
+//
+
+import UIKit
+
+final class ReusableViewText: UICollectionReusableView {
+
+    static let identifier = "ReusableViewTextIdentifier"
```
Comment (filipealva) [inline Rocket.Chat/Views/Cells/Preferences/ReusableViewText.swift]: Nothing specifically related to this PR, just a note: what if we start using `String(describing: Self.self)` for identifiers like this one in the future?

Comment (rafaelks) [reply Rocket.Chat/Views/Cells/Preferences/ReusableViewText.swift]: Sounds good to me... could even be part of a Generic? 🤔

Comment (filipealva) [reply Rocket.Chat/Views/Cells/Preferences/ReusableViewText.swift]: Cool. Yes, it could :D

**Pull request merged**